src/test/cypress/cypress/integration/GeoMap.spec.js

```diff
@@ -0,0 +1,175 @@
+            script.setPropertyInConf('feed.geomap.enableMap', 'web-ui', true);
+            opfab.loginWithUser('operator1_fr');
+            script.sendCard('cypress/geoMap/message1.json');
+            cy.get('[id^="opfab-feed-light-card-cypress-message1"]').find('em.fa.fa-location-crosshairs').should('not.exist');
+            cy.get('of-map').click();
+            cy.get('[id^="opfab-feed-light-card-cypress-message1"]').should('have.length', 1);
+        });
+
+        it('Card with invalid polygon data is not pointed on the geomap', () => {
+            script.setPropertyInConf('feed.geomap.enableMap', 'web-ui', true);
+            opfab.loginWithUser('operator1_fr');
+            script.sendCard('cypress/geoMap/invalid_polygon_card1.json');
+            cy.get('[id^="opfab-feed-light-card-cypress-invalid_polygon_card1"]').find('em.fa.fa-location-crosshairs').click();
+            cy.get('of-map').click();
+            cy.get('[id^="opfab-feed-light-card-cypress-invalid_polygon_card1"]').should('have.length', 1);
+        });
+    });
+
+    describe('Check if crosshair icon is only visible on cards with a geo-location', function () {
+        it('Card with geo-location data should have the crosshair icon', () => {
+            script.setPropertyInConf('feed.geomap.enableMap', 'web-ui', true);
+            opfab.loginWithUser('operator1_fr');
+            script.sendCard('cypress/geoMap/point_card1.json');
+            cy.get('[id^="opfab-feed-light-card-cypress-point_card1"]').find('em.fa.fa-location-crosshairs').should('exist');
+        });
+
+        it('Card without geo-location data should not have the crosshair icon', () => {
+            script.setPropertyInConf('feed.geomap.enableMap', 'web-ui', true);
+            opfab.loginWithUser('operator1_fr');
+            script.sendCard('cypress/geoMap/message1.json');
+            cy.get('[id^="opfab-feed-light-card-cypress-message1"]').find('em.fa.fa-location-crosshairs').should('not.exist');
+        });
+
+        it('Card with invalid geo-location data shoudl have the crosshair icon', () => {
+            script.setPropertyInConf('feed.geomap.enableMap', 'web-ui', true);
+            opfab.loginWithUser('operator1_fr');
+            script.sendCard('cypress/geoMap/invalid_point_card1.json');
+            cy.get('[id^="opfab-feed-light-card-cypress-invalid_point_card1"]').find('em.fa.fa-location-crosshairs').should('exist');
+        })
+    });
+
+    describe('Zoom-to-location works', function () {
+        it('Clicking on the crosshair icon will zoom to the location in the map', () => {
+            script.setPropertyInConf('feed.geomap.enableMap', 'web-ui', true);
+            opfab.loginWithUser('operator1_fr');
+            script.sendCard('cypress/geoMap/point_card1.json');
+            cy.get('[id^="opfab-feed-light-card-cypress-point_card1"]').find('em.fa.fa-location-crosshairs').click();
+        });
+    });
```

Comment:
This test does not seems useful for me as it has already been test in `Card with point data is pointed on the geomap`

Reply:
That is true. I couldn't find a way to check if the map is actually zooming in. I will remove this one.